index.html

```diff
@@ -7,12 +7,59 @@
     <link href='//fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
     <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
     <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="style.css">
     <!-- Add a link to your CSS file here (use the line above to guide you) -->
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
+  <nav class="navigation"> 
+    <div>
+      <ul class= "links">
+        <li class="current-page"><a href="">Meet Karma</a></li>
+        <li><a href="">How It Works</a><li>
+        <li><a href="">Store</a></li>
+        <li><a href="">Blog</a></li>
+        <li><a href="">Help</a></li>
+        <li><a href="">Login</a></li>
+      </ul>
+      <img class="logo" src="img/karma-logo.svg" alt="Logo Image" width="26px" height="70px">
```

Comment:
Be careful with including attributes like width and height as part of the img element - these are best included as part of the class so that styling is all in one place.